create_marc_record.py

```diff
@@ -0,0 +1,136 @@
+from pymarc import Record, Field, Subfield
+
+
+def create_base_record() -> Record:
+    """Create a base MARC record, to which metadata from an external
+    source will be added."""
+    record = Record()
+
+    # Leader fixed field.
+    # Type (Leader/06) - j (Musical sound recording)
+    record.leader.type_of_record = "j"
+    # BLvl (Leader/07) - m (Monographic/Item)
+    record.leader.bibliographic_level = "m"
+    # Ctrl (Leader/08) - # (No specified type)
+    record.leader.type_of_control = "#"
+    # ELvl (Leader/17) - 3 (Abbreviated level)
+    record.leader.encoding_level = "3"
+    # Desc (Leader/18) - i (ISBD punctuation included)
+    record.leader.cataloging_form = "i"
+
+    # 007 - physical description fixed field
+    # 007 ## $a s $b d $d f $e u $f n $g g $h n $i n $m e $n u
+    # translated to fixed field:
+    record.add_field(Field(tag="007", data="sd#fungnn###eu"))
+
+    # 008 - general information fixed field
+    # DtSt (008/06) - s (Single known date/probable date)
+    # Ctry (008/15-17) - xx# (No place, unknown, or undetermined)
+    # Comp (008/18-19) - ##
+    # FMus (008/20) - n (Not applicable)
+    # Part (008/21) - n (Not applicable)
+    # Audn (008/22) - # (Unknown or unspecified)
+    # Form (008/23) - # (None of the following)
+    # AccM (008/24-29) - # (No accompanying matter)
+    # LTxt (008/30-31) - # (Item is a music sound recording)
+    # TrAr (008/33) - # Not arrangement or transposition or not specified
+    # MRec (008/38) - # (Not modified)
+    # Srce (008/39) - d (Other)
+    record.add_field(Field(tag="008", data="000000s00000000xx###nn#################d"))
```

Comment:
In this 008, Hermine's example has `008  240322s||||\\\\xx\\\nn\\\\\\\\\\\n\|||\d`.

The `\` represent blanks, so change all the `#` to blanks.

008/07-10 (Date 1) has fill characters, `|` (no attempt to code).  008/11-14 (Date 2) should be blanks (not applicable, since 008/06 = `s`, single date).

008/17 should be blank.
008/18-19 should be fill characters, I think, though Hermine has blanks - but I don't see those as valid in the docs.
008/22, 008/23, 008/24-29, 008/30-31, 008/32: all blanks
008/33: Hermine's specs have blank, but her email template has `n` (Not applicable).  That's also what we used in the past.  So please ask her about that difference.
008/34: blank
008/35-37: Hermine's email template has fill characters for the language.  In the past we used `zxx` (No linguistic content), but fills seem equally valid, so please use those.
008/38: blank

Also, for 008/00-05, we'll need to fill in the current YYMMDD when this runs, similar to data fields where a date is used.  So as a reminder placeholder, `YYMMDD` probably is better than `000000`.

https://www.loc.gov/marc/bibliographic/bd008a.html (certain positions in all 008s) and
https://www.loc.gov/marc/bibliographic/bd008m.html (the other positions for the music format).